application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php

```diff
@@ -29,12 +29,12 @@
                 code
               },
               success: (data) => {
-                if (data) {
-                    $('#question-title-warning').text(data);
-                    $('#question-title-warning').removeClass('hidden');
-                } else {
-                    $("#submit-copy-question").click();
-                }
+                  if (data.hasOwnProperty('message') && data.message === null) {
```

Comment:
Didn't dig a lot into the issue, but just smell it... Shold the condition be like this maybe?
`!data.hasOwnProperty('message') || data.message === null`

Reply:
According to changes on BE side, message item is mandatory. That's why I checking for item existence with null value. For me it seems like anti-pattern too. But we already have these changes in BE. WDYT?

application/controllers/QuestionAdministrationController.php:3036

![image](https://user-images.githubusercontent.com/11578023/208687166-94cf778a-2709-4fd0-be07-8a866fba983e.png)

Reply:
So if there is no message property, data will be just showed as before. Also, this is already on develop? Seems OK then :)

Reply:
I'm not sure I got you correctly. Message property will be still permanently in response. And acceptance criteria to have a null as value (according to BE code). And yeah, it's already fixed in dev branch ✅